test/router/v2/integration/SynapseRouterV2.Integration.Avalanche.t.sol

```diff
@@ -0,0 +1,981 @@
+    address private constant AVAX_SYN_ROUTER_V1 = 0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a;
+    address private constant AVAX_SYN_BRIDGE = 0xC05e61d0E7a63D27546389B7aD62FdFf5A91aACE;
+    address private constant AVAX_SYN_CCTP = 0xfB2Bfc368a7edfD51aa2cbEC513ad50edEa74E84;
+    address private constant AVAX_STABLE_POOL = 0xED2a7edd7413021d440b09D654f3b87712abAB66;
+
+    // bridge tokens
+    address private constant NUSD = 0xCFc37A6AB183dd4aED08C204D1c2773c0b1BDf46;
+    address private constant SYN = 0x1f1E7c893855525b303f99bDF5c3c05Be09ca251;
+    address private constant NETH = 0x19E1ae0eE35c0404f835521146206595d37981ae;
+    address private constant WSOHM = 0x240E332Cd26AaE10622B24160D23425A17256F5d;
+    address private constant NFD = 0xf1293574EE43950E7a8c9F1005Ff097A9A713959;
+    address private constant GOHM = 0x321E7092a180BB43555132ec53AaA65a5bF84251;
+    address private constant WAVAX = 0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7;
+    address private constant GMX = 0x62edc0692BD897D2295872a9FFCac5425011c661;
+    address private constant UST = 0xE97097dE8d6A17Be3c39d53AE63347706dCf8f43;
+    address private constant NEWO = 0x4Bfc90322dD638F81F034517359BD447f8E0235a;
+    address private constant BTC_B = 0x152b9d0FdC40C096757F570A51E494bd4b943E50;
+    address private constant SDT = 0xCCBf7c451F81752F7d2237F2c18C371E6e089E69;
+    address private constant JEWEL = 0x997Ddaa07d716995DE90577C123Db411584E5E46;
+    address private constant SFI = 0xc2Bf0A1f7D8Da50D608bc96CF701110d4A438312;
+    address private constant H2O = 0xC6b11a4Fd833d1117E9D312c02865647cd961107;
+    address private constant USDC = 0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E;
+
+    // supported tokens (for adapter swaps)
+    address private constant USDC_E = 0xA7D7079b0FEaD91F3e65f86E8915Cb59c1a4C664;
+    address private constant DAI_E = 0xd586E7F844cEa2F87f50152665BCbc2C279D8d70;
+    address private constant USDT_E = 0xc7198437980c041c805A1EDcbA50c1Ce5db95118;
+    address private constant WETH_E = 0x49D5c2BdFfac6CE2BFdB6640F4F80f226bc10bAB;
+    address private constant USDT = 0x9702230A8Ea53601f5cD2dc00fDBc13d4dF4A8c7;
+    address private constant GMX_WRAPPER = 0x20A9DC684B4d0407EF8C9A302BEAaA18ee15F656;
+
+    constructor() SynapseRouterV2IntegrationTest("avalanche", AVAX_BLOCK_NUMBER, AVAX_SWAP_QUOTER) {}
+
+    function afterBlockchainForked() public virtual override {
+        synapseLocalBridgeConfig = AVAX_SYN_ROUTER_V1;
+        synapseBridge = AVAX_SYN_BRIDGE;
+        synapseCCTP = AVAX_SYN_CCTP;
+    }
+
+    function addExpectedTokens() public virtual override {
+        addExpectedToken(NUSD, "nUSD");
+        addExpectedToken(USDC_E, "USDC.e");
+        addExpectedToken(DAI_E, "DAI.e");
+        addExpectedToken(USDT_E, "USDT.e");
+        addExpectedToken(USDC, "CCTP.USDC");
+        addExpectedToken(NETH, "nETH");
+        addExpectedToken(WETH_E, "WETH.e");
+        addExpectedToken(WAVAX, "WAVAX");
+        addExpectedToken(GMX, "GMX");
```

Comment:
Native USDT is missing here - my understanding is that we have all the major tokens to test here?

Reply:
In terms of stables and ETH, yes that's the thought but let me know if you want me to add more besides USDT native

Reply:
I think that this list should cover all the important cases:
- "supported" stablecoins (ones swappable to nUSD and/or USDC)
- nETH, WETH
- Wrapped Gas token (if not ETH)
- Any edge case chain-specific tokens (like GMX on Avalanche, but on on Arbitrum)